### PR TITLE
Introduce per-environment JWKS paths

### PR DESCRIPTION
**Pull request by joshgarde**
Urgent hotfix to address authentication issues caused by EDL endpoint changes by introducing per-environment JWKS paths

Closes https://github.com/podaac/swodlr/issues/147

### CODE REVIEW

File: terraform/environments/sit.env [hunk 1]
```diff
@@ -4,3 +4,4 @@ export BUCKET=podaac-services-sit-terraform
 export TF_VAR_container_image_tag=develop
 export TF_VAR_active_profiles="[\"dev\"]"
 export TF_VAR_tea_mapping="{\"podaac-swot-sit-swodlr-protected\"=\"archive.swot.podaac.sit.earthdata.nasa.gov\"}"
+export TF_VAR_edl_jwks_path=".well-known/edl_ops_jwks.json"
```
Comment (frankinspace) [inline terraform/environments/sit.env:7]: should this be 
```suggestion
export TF_VAR_edl_jwks_path=".well-known/edl_sit_jwks.json"
```

File: terraform/environments/uat.env [hunk 2]
```diff
@@ -4,3 +4,4 @@ export BUCKET=podaac-services-uat-terraform
 export TF_VAR_container_image_tag=develop
 export TF_VAR_active_profiles="[\"\"]"
 export TF_VAR_tea_mapping="{\"podaac-swot-uat-swodlr-protected\"=\"archive.swot.podaac.uat.earthdata.nasa.gov\"}"
+export TF_VAR_edl_jwks_path=".well-known/edl_ops_jwks.json"
```
Comment (frankinspace) [inline terraform/environments/uat.env:7]: should this be
```suggestion
export TF_VAR_edl_jwks_path=".well-known/edl_uat_jwks.json"
```

Comment (joshgarde) [reply terraform/environments/uat.env:7]: Every environment currently authenticates on OPS

**Comment by frankinspace** [comment]
Since we need to get this to ops pretty quick I'm going to do a patch release

**Review by frankinspace — APPROVED** [review]

**Pull request merged**